$NetBSD$
pkgsrc doesn't have lib64 or targetdir so don't set it.
fixes build on linux.
http://sf.net/p/qore/code/6817/
--- configure.orig	2014-11-01 13:45:54.822586579 +0000
+++ configure
@@ -15674,14 +15674,6 @@ if test "$enable_64bit" = "yes"; then
       *linux*)	if test "$GXX" = "yes"; then
 		   CXXFLAGS="$CXXFLAGS -m64"
 		fi
-		# only set LIBSUFFIX=64 if /usr/lib64 exists
-		if test -d /usr/lib64; then
-		   LIBSUFFIX=64
-                fi
-		target="${host_cpu}-${host_os}"
-                if test -d /usr/lib/$target; then
-                   LIBSUFFIX=/$target
-                fi
 
 		;;
       *hpux*)	if test "$host_cpu" = "ia64"; then
